### DISCUSSION TITLE
Announcing Sionna v0.12

### DISCUSSION THREAD

**Discussion by faycalaa** [NVlabs/sionna#74] [Announcements]
We are happy to announce the Sionna v0.12 release:

- MIMO
    - Add `MMSEPICDetector` layer for MMSE-PIC detection, contribution from @rwiesmayr
    - Add `EPDetector` layer for expectation propagation (EP) detection
    - Add `KBestDetector` for K-Best detection
    - Utility layer `PAM2QAM` now also supports soft information
- FEC
     - Add `OSDDecoder` layer for ordered statistics decoding (OSD)
     - Add submodule `fec.linear` for universal support of Linear Codes
     - Add support for max-log BCJR decoding to the convolutional and Turbo decoders
     - Align Turbo and convolutional encoders
     - Add support for code termination to convolutional codes
     - Add input LLR clipping to LDPC BP decoding
- OFDM
    - Add base class `BaseChannelEstimator` to ease the implementation of OFDM channel estimators
    - Add base class `BaseChannelInterpolator` to ease the implementation of OFDM channel interpolators
    - Add `LMMSEInterpolator` for LMMSE channel time and frequency interpolation and optional spatial smoothing
    - Add utility functions to compute time and frequency covariance matrices of TDL models
    - Add base class `BaseEqualizer` to ease implementation of OFDM MIMO equalizers
    - Add base classes `BaseDetector` and `BaseDetectorWithPrior` to ease the implementation of OFDM MIMO detectors without and with prior information
    - Add `LinearDetector` layer for LMMSE, MR, or ZF equalization followed by app or max-log demapping for OFDM MIMO detection
    - Add `KBestDetector` for K-Best-based OFDM MIMO detection
    - Add `EPDetector` for EP-based OFDM MIMO detection
    - Add `MMSEPICDetector` for MMSE-PIC-based OFDM MIMO detection
    - Add a setter to `PilotPattern` to support one-the-fly update of pilot symbols
- Channel
    - Add MIMO support to TDL channel models through the specification of arbitrary correlation matrices
    - Add support for TDL A30, B100, and C300 channel models
    - Fix base station orientation in `gen_topology_*` utilities
- Tutorials
    - Add a notebook showing how to use and compare some of the OFDM channel estimators and MIMO detectors available in Sionna
- MISC
    - Typos fix and new test cases
    - Drop support for TensorFlow 2.6

**Comment by SeeseeU** [comment]
您的邮件已经成功接收，仅以这封回信心表达我的荣幸和对您诚挚的祝福。
